torch/utils/flop_counter.py

```diff
@@ -722,8 +724,34 @@ def __exit__(self, *args):
 
     def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         kwargs = kwargs if kwargs else {}
-        out = func(*args, **kwargs)
-        return self._count_flops(func._overloadpacket, out, args, kwargs)
+
+        # Skip non-decomposoable ops from non-standard dispatch_sizes_strides_policy such as NJT
+        if func in [torch.ops.aten.is_contiguous.default,
+            torch.ops.aten.is_contiguous.memory_format,
+            torch.ops.aten.is_strides_like_format.default,
+            torch.ops.aten.is_non_overlapping_and_dense.default,
+            torch.ops.aten.size.default,
+            torch.ops.aten.sym_size.default,
+            torch.ops.aten.stride.default,
+            torch.ops.aten.sym_stride.default,
+            torch.ops.aten.storage_offset.default,
+            torch.ops.aten.sym_storage_offset.default,
+            torch.ops.aten.numel.default,
+            torch.ops.aten.sym_numel.default,
+            torch.ops.aten.dim.default, 
+            torch.ops.prim.layout.default]:
```

Comment:
this list is long enough that you should consider shoving it in a set and memoizing it

Reply:
@jbschlosser is anything missing from the list? The values come from https://github.com/pytorch/pytorch/blob/b9f0563aafe9e21e71f5da6d7a6ffd62d9403641/torch/nested/_internal/ops.py#L414:L425 + `prim::layout` which was needed to pass the tests on my machine.

Reply:
LGTM, thanks!